library/src/main/java/com/google/maps/android/StreetViewUtil.kt

```diff
@@ -35,14 +35,24 @@ class StreetViewUtils {
         /**
          * This function will check whether a location is available on StreetView or not.
          *
-         * @param latLng Location to check
-         * @param apiKey Maps API Key
-         * @return A Status value specifying if the location is available on Street View or not,
```

Comment:
Should revert the line that deletes the docs for the return value.

Reply:
Done ✅